service/header/subscription.go

```diff
@@ -0,0 +1,56 @@
+package header
+
+import (
+	"context"
+
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+)
+
+// ExtendedHeaderSubTopic hardcodes the name of the ExtendedHeader
+// gossipsub topic.
+const ExtendedHeaderSubTopic = "header-sub"
+
+// subscription handles retrieving ExtendedHeaders from the header pubsub topic.
+type subscription struct {
+	topic        *pubsub.Topic
+	subscription *pubsub.Subscription
+}
+
+// newSubscription creates a new ExtendedHeader event subscription
+// on the given host.
+func newSubscription(topic *pubsub.Topic) (*subscription, error) {
```

Comment:
Hmm. It's not clear, but where in this function is specifically `ExtendedHeaderSubTopic` used?

Reply:
`ExtendedHeaderSubTopic` is used in header/service.go, specifically in the code that creates an instance of the *pubsub.Topic.